roles/rsyslog/defaults/main.yaml

```diff
@@ -948,36 +961,51 @@ rsyslog__conf_viaq_elasticsearch:
               } else {
                   set $.viaq_index_name = exec_template("prefix_index_template");
               }
-          {% if openshift_logging_use_ops | default(False) %}
-              if $.viaq_index_prefix startswith "project." then {
+          {% for res in rsyslog_elasticsearch %}
+          {% if loop.length == 1 %}
+              set $.es_host="{{ res.server_host | default('logging-es') }}"
+              set $.es_port="{{ res.server_port | default(9200) }}"
+            {% if use_omelasticsearch_cert | default(False) %}
+              set $.es_ca_cert="{{ res.ca_cert }}"
+              set $.es_cert="{{ res.cert }}"
+              set $.es_key="{{ res.key }}"
+            {% endif %}
+          {% else %}
+          {% if loop.first %}
```

Comment:
Can we remove the above  {% if loop.length == 1 %}  section from lines 965 till 973 and instead update the loop conditions in this line (974) and line 983 to:

{% if loop.first and  loop.length > 1  %}
{% elif loop.last and  loop.length > 1 %}

?

@nhosoi @richm